packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -677,6 +695,27 @@ export class MathExpression implements IMetric {
     });
   }
 
+  /**
+   * Make a new AnomalyDetectionAlarm for this metric
+   *
+   * Combines both properties that may adjust the metric (aggregation) as well
+   * as alarm properties.
+   */
+  public createAnomalyDetectionAlarm(scope: Construct, id: string, props: CreateAnomalyDetectionAlarmOptions): AnomalyDetectionAlarm {
+    return new AnomalyDetectionAlarm(scope, id, {
+      metric: this,
+      alarmName: props.alarmName,
+      alarmDescription: props.alarmDescription,
+      comparisonOperator: props.comparisonOperator,
+      datapointsToAlarm: props.datapointsToAlarm,
+      threshold: props.threshold,
+      evaluationPeriods: props.evaluationPeriods,
+      treatMissingData: props.treatMissingData,
+      actionsEnabled: props.actionsEnabled,
```

Comment:
We should be able to just send `...props` here:

```ts
return new AnomalyDetectionAlarm(scope, id, {
  metric: this,
  ...props,
});

Reply:
Fixed